lib/rds_sql_server/sql_server_base.rb

```diff
@@ -1,7 +1,7 @@
 class SqlServerBase < ActiveRecord::Base
   begin
     rds = Rds.new
-    cert_path = ENV['RDS_CERT_PATH'].presence || '/etc/ssl/certs/rds-combined-ca-bundle.pem'
+    cert_path = ENV['RDS_CERT_PATH'].presence || '/etc/ssl/certs/us-east-1-bundle.pem'
```

Comment:
Might be worth testing this out, but I'm not sure how to do that on QA.

Reply:
Tested.  Works.  Thanks!